src/services/sync-component-with-file/sync-component.test.ts

```diff
@@ -33,7 +33,7 @@ const getMockedComponent = (overrideMockedComponent: Partial<Component> = {}): C
   id: 'id',
   name: 'mock',
   type: CompassComponentType.Service,
-  typeId: 'service',
+  typeId: 'SERVICE',
```

Comment:
NIT: use `DEFAULT_COMPONENT_TYPE_ID` instead of 'SERVICE'

Reply:
Considering that If the default component type id is modified, this test case will fail because the ENUM is hardcoded to SERVICE. 
When the Enum is deprecated completely, I would recommend using the DEFAULT COMPONENT TYPE ID